DigitalLearningSolutions.Web/Helpers/FilteringHelper.cs

```diff
@@ -9,13 +9,34 @@
     public static class FilteringHelper
     {
         public const char Separator = '|';
+        public const char FilterSeparator = '╡';
+
+        public static string? AddNewFilterToFilterBy(string? filterBy, string? filterValue)
+        {
+            if (filterValue != null)
+            {
+                if (filterBy != null)
+                {
+                    if (!filterBy.Contains(filterValue))

```

Comment:
In what situations do we expect there to be a null filterBy and a non-null filterValue?

Reply:
First filter being added